include/perf_watcher.h

```diff
@@ -47,18 +49,24 @@ typedef struct PerfWatcher {
   X(NOCOUNT, "nocount", nocount, NOCOUNT)                                      \
   X(TRACEPOINT, "tracepoint", events, NOCOUNT)                                 \
   X(CPU_NANOS, "cpu-time", nanoseconds, CPU_SAMPLE)                            \
-  X(CPU_SAMPLE, "cpu-sample", count, NOCOUNT)
+  X(CPU_SAMPLE, "cpu-samples", count, NOCOUNT)                                 \
+  X(ALLOC_SAMPLE, "alloc-samples", count, NOCOUNT)                             \
+  X(ALLOC_SPACE, "alloc-space", bytes, ALLOC_SAMPLE)
```

Comment:
this works well with David's work

Reply:
Yeah, some of them are deprecated, but I included them for the sake of completeness.